### PR TITLE
Vrm examples

### PR DESCRIPTION
**Pull request by dccowan**
Addresses issues in #714

### CODE REVIEW

**Comment by dccowan** [comment]
The purpose of this pull request was to:

- Reduce the length of current VRM example scripts to a manageable level
- Replace assert error management with raise error
- Reduce the time it takes to form the sensitivity matrix for VRM problems.

File: SimPEG/VRM/RxVRM.py [hunk 1]
```diff
@@ -5,12 +5,15 @@
...
+
 class BaseRxVRM(Survey.BaseRx, properties.HasProperties):
     """Base VRM receiver class"""
 
     def __init__(self, locs, **kwargs):
 
-        super(BaseRxVRM, self).__init__(locs, 'None', storeProjections=False, **kwargs)
+        super(BaseRxVRM, self).__init__(
+            locs, 'None', storeProjections=False, **kwargs
+            )
```
Comment (lheagy) [inline SimPEG/VRM/RxVRM.py]: This is nit-picky, but would you mind closing the bracket at the same level as the `super` (eg back one indentation)

File: SimPEG/VRM/RxVRM.py [hunk 2]
```diff
@@ -21,12 +24,19 @@ class Point(BaseRxVRM):
     """Point receiver"""
 
     times = properties.Array('Observation times', dtype=float)
-    fieldType = properties.StringChoice('Field type', choices=["h", "b", "dhdt", "dbdt"])
-    fieldComp = properties.StringChoice('Component of response', choices=["x", "y", "z"])
+    fieldType = properties.StringChoice(
+        'Field type', choices=["h", "b", "dhdt", "dbdt"]
+        )
```
Comment (lheagy) [inline SimPEG/VRM/RxVRM.py]: same here

File: SimPEG/VRM/SurveyVRM.py [hunk 3]
```diff
@@ -35,8 +35,9 @@ def __init__(self, srcList, **kwargs):
 
     @properties.validator('t_active')
     def _t_active_validator(self, change):
-        assert self.nD == len(change['value']), (
-            "Length of t_active boolean array must equal number of data")
+        if self.nD != len(change['value']):
+            raise ValueError(
+                'Length of t_active boolean array must equal number of data. Number of data is %i' % self.nD)
```
Comment (lheagy) [inline SimPEG/VRM/SurveyVRM.py]: would you mind putting the closing bracket on a new line

**Comment by lheagy** [comment]
Thanks @dccowan: this looks good! Would you mind adding a few notes so I can use them for the release notes? Just a couple statements on what you changed in the code and additions you made for the examples. If you would like to upload an image, I can include that too!

**Pull request merged**